src/app/model/teacher.ts

```diff
@@ -3,17 +3,23 @@ export class Teacher {
   static UPDATED_EVENT = 'teacher-updated';
   static DELETED_EVENT = 'teacher-deleted';
 
-  id: string
+  id: string;
   firstName: string;
-  surname: string;
+  surName: string;
```

Comment:
Helaas: het moet toch echt surname blijven. Het is één woord, en geen samenstelling.